### DISCUSSION TITLE
Start an app at startup placing it on specific workspace, without persistent rules.

### DISCUSSION THREAD

**Discussion by lukolszewski** [i3/i3#5563] [Q&A]
My questions is if this is possible in i3 (without having to implement new functionality). So, let me get to the point right away.

I'm a long term gnome user just giving i3 a serious try and one of the things I'm trying to set up is to have a number of apps started on their respective workspaces on startup. For some apps that run a single instance that will always live on a single workspace I can set a for_window or assign and that app will always be on that workspace (example keepassxc). For xterm that I know I will always have 2 instances I can set window titles and move them to specific workspaces based on that. 

But then enters chrome (or any other browser). I'd like to have an instance of chrome start on 2nd workspace, but Id like to retain the ability to launch another instance (or window) of chrome and manually move it to another instance without touching the previous one and that proves difficult to achieve.

If I set for_window on the window class, all chrome windows will be on that one workspace. I can't use a title, because the moment the title changes the window will jump to workspace 1... I briefly considered various timing shenaningans for example trying to wait a second after all other apps have started, switch the workspace to no 2, wait another second then just launch chrome, but it didn't work as expected also it seems a prytty janky way to do it.

So, is there a solution to this? Or am I going to have to start(or move) chrome manually on every startup if I'd like this functionality in i3?

**Comment by Airblader** [comment]
Have you seen this?

https://i3wm.org/docs/layout-saving.html

**Reply by lukolszewski** [reply]
I haven't thanks.
